explorer/src/lib/components/__tests__/__snapshots__/Navbar.spec.js.snap

```diff
@@ -44,7 +44,7 @@ exports[`Navbar > renders the Navbar component 1`] = `
     <span
       class="dusk-badge dusk-badge--variant-success dusk-navbar__menu--network"
     >
-      Local
+      nodes.dusk.network
```

Comment:
Is this what we want to display in the Navbar badge? I am asking because I do not know if we are moving away from `Local, Testnet, Devnet, Mainnet`.

Reply:
I was considering taking the first part of the domain name, perhaps that is better now that the network names are no longer configured.